source/releasenotes/024-05-06-domain-validation.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Domain validation required for all sites "
+published_date: "2024-05-06"
+categories: [security, infrastructure, action-required]
+---
+
+As part of our continued effort to protect sites on our platform, Pantheon now requires all sites on the platform to prove domain ownership via TXT records. This enhances security by ensuring domain ownership and protection, safeguarding against domain hijacking by malicious actors. 
```

Comment:
```suggestion
As part of our continued effort to protect sites on our platform, Pantheon now requires all sites on the platform to prove domain ownership via TXT records.
This verification is required before Pantheon will accept traffic on a domain name for a given site.
This change enhances security by preventing a malicious actor from bringing traffic to a newly created site on Pantheon using a domain that pointed to a since-deleted site on Pantheon.
```